actions.yaml

```diff
@@ -0,0 +1,2 @@
+get-profile-a-home-network-public-key:
```

Comment:
I'm not sure about the need for this action. Please convince me it will be actually used.

Reply:
So like this we make sure that the key is there for the `udm` to get the SUPI for any UE that used the public key to create a SUCI, and we have a way of fetching that key for UEs to use it.

Reply:
In this case, I would wait for us to actually need the action. It's possible that we need it very soon or never but I don't want to have this code "just in case".

Reply:
To run the simulation you don't really need the key pair at all, neither the private key nor the public one, I tested the deployment and the simulation while removing the key from the config and it works. But that's the simulation using `GNBSIM`

However why couldn't someone make use of the key pair right now? deploy SDCORE, configure UEs and add them? How do we distribute the public key to them otherwise? so for me it's not clear why would it be a "just in case" scenario.

My point is, if we are adding the private key to the configuration we need to distribute the public key somehow to be able to configure the UEs otherwise the private key itself has no added value, the Juju action might not be the only way to do this, but it's better than not having any mechanism to do that.

Please let me know if I'm missing anything here :) @gruyaume